json-schema/json-schema/src/main/scala/endpoints/algebra/JsonSchemas.scala

```diff
@@ -145,3 +145,8 @@ trait JsonSchemas {
   ): JsonSchema[C[A]]
 
 }
+
+object JsonSchemas {
+
+  val defaultDiscriminatorName: String = "type"
```

Comment:
Given the scope of `Tagged`, it would be fine to get this back into `trait JsonSchemas` with current name so that the user can set algebra-wise default name as well as each schema specific value.

Reply:
(sorry for somewhat inconsistent feedback, I had no idea of the default name encoding at first.)